native/js/HappoJob.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component, PropTypes } from 'react';
+import {
+  View,
+} from 'react-native';
+import io from 'socket.io-client';
+import StoryRenderer from './StoryRenderer';
+import { getStories, getStory } from './StoryManager';
```

Comment:
I like the name "story". Assuming it's the same thing that we use for "example" in the js version, I could consider changing it globally to "story".